docs/getting-started.md

```diff
@@ -161,6 +161,11 @@ please refer to this [guide](kind.md).
 To deploy Antrea in a [Minikube](https://github.com/kubernetes/minikube) cluster,
 please refer to this [guide](minikube.md).
 
+### Deploying Antrea in Rancher Cluster
```

Comment:
Can we omit "Cluster" here?
We don't use "Cluster" with Kind (L154) and Minikube (L159).

Reply:
here we need to use the word Cluster, because Rancher is a cluster manager here.

Reply:
It is better to use Deploying Antrea in Rancher Managed Cluster